common/src/main/java/earth/terrarium/prometheus/common/commands/utilities/HomeCommand.java

```diff
@@ -60,7 +60,10 @@ private static LiteralArgumentBuilder<CommandSourceStack> add() {
         return Commands.literal("sethome")
             .then(Commands.argument("name", StringArgumentType.greedyString())
                 .executes(context -> {
-                    HomeHandler.add(context.getSource().getPlayerOrException(), StringArgumentType.getString(context, "name"));
+                    ServerPlayer player = context.getSource().getPlayerOrException();
+                    String homeName = StringArgumentType.getString(context, "name");
+                    HomeHandler.add(player, homeName);
```

Comment:
this need to return if add returns false 
```suggestion
                    if (!HomeHandler.add(player, homeName)) return 0;
```